tests/test_env.py

```diff
@@ -222,6 +222,7 @@ def test_rio_env_credentials_options(tmpdir, monkeypatch, runner):
                            "aws_secret_access_key = bar\n"
                            "aws_session_token = baz")
     monkeypatch.setenv('AWS_SHARED_CREDENTIALS_FILE', str(credentials_file))
+    monkeypatch.setenv('AWS_SESSION_TOKEN', 'ignore_me')
```

Comment:
@jacquestardie I've got a test to make sure that command line option `--aws-profile` has higher precedence than environment variables.